code/modules/food&drinks/recipes/drinks_recipes.dm

```diff
@@ -364,7 +364,7 @@
 	name = "Mead"
 	id = "mead"
 	result = "mead"
-	required_reagents = list("sugar" = 1, "water" = 1)
+	required_reagents = list("honey" = 2)
```

Comment:
You don't see it there because I didn't change it, but right below it is a thing saying 'CATALYST : ENYZME.'